src/vsr/superblock.zig

```diff
@@ -320,23 +316,38 @@ pub const SuperBlockHeader = extern struct {
         manifest_newest_checksum_padding: u128 = 0,
         snapshots_block_checksum: u128,
         snapshots_block_checksum_padding: u128 = 0,
+        free_set_last_block_checksum: u128,
+        free_set_last_block_checksum_padding: u128 = 0,
 
         manifest_oldest_address: u64,
         manifest_newest_address: u64,
         snapshots_block_address: u64,
+        free_set_last_block_address: u64,
```

Comment:
Very much a nit, but maybe "free set" could precede "manifest" in this order, since it is accessed sooner, and is "lower level" (grid rather than LSM).

Reply:
:+1: that's actually exactly the reason why I put FreeSetNode before ManifestNode in the scheme!